README.md

```diff
@@ -67,6 +67,8 @@ Usage
       - MIPS
       - PowerPC
       - Sparc
+      - RISC-V 64
+      - RISC-V Compressed
```

Comment:
Can you add the same in [ropgadget/args.py](https://github.com/JonathanSalwan/ROPgadget/blob/master/ropgadget/args.py#L48)?

Reply:
Done.